sqlmesh/core/engine_adapter/databricks.py

```diff
@@ -280,7 +280,10 @@ def _build_table_properties_exp(
             table_kind=table_kind,
         )
         if clustered_by:
-            clustered_by_exp = exp.Cluster(expressions=[exp.column(col) for col in clustered_by])
+            cluster_key = exp.maybe_parse(
+                f"({','.join(clustered_by)})", into=exp.Ordered, dialect="databricks"
```

Comment:
nit: you could use `dialect=self.dialect` here instead of hardcoding `"databricks"`

Reply:
yeah thanks for calling it out, it was just me being lazy :P